src/main/drivers/dma_reqmap.c

```diff
@@ -250,6 +403,18 @@ dmaoptValue_t dmaoptByTag(ioTag_t ioTag)
         }
     }
 #else
+#if defined(STM32H7)
+    const timerHardware_t *timhw = timerGetByTag(ioTag);
+    if (timhw && timhw->dmaRefConfigured && timhw->dmaChannelConfigured) {
+        // Reverse lookup dmaopt from dmaRefConfigured by utilizing dmaChannelSpec array
+        unsigned opt;
+        for (opt = 0; opt < ARRAYLEN(dmaChannelSpec); opt++) {
+            if (timhw->dmaRefConfigured == dmaChannelSpec[opt].ref) {
+                return (dmaoptValue_t)opt;
```

Comment:
Not sure that `dmaoptByTag()` should return anything but `DMA_OPT_UNUSED` for H7 if timer management is not defined. Otherwise users will try and change DMA options where this is not possible.

Reply:
Ah, I see. I was partly confused by USE_TIMER_MGMT and USE_DMA_SPEC dependency.

Reply:
The thing is that although `USE_DMA_SPEC` can be enabled without `USE_TIMER_MGMT`, custom DMA settings can only be persisted if `USE_TIMER_MGMT` is enabled as well, as the settings are persisted in `timerIOConfig`.
We could probably rewrite it so that `USE_DMA_SPEC` could persist stuff on its own, but when I did it I assumed that there would be little demand for what is essentially half a solution for Unified targets, so I didn't bother.

The only value I see in having separate defines for `USE_DMA_SPEC` and `USE_TIMER_MGMT` is for transitional states, to reduce the amount of change needed per pull request when adding full configurability, like what you are doing in this pull request - I don't think we'll ever want to release 'half unified targets' that had only DMA or only timers configurable.

Reply:
> I don't think we'll ever want to release 'half unified targets' that had only DMA or only timers configurable.

👍 

Code already modified and pushed.